helm/prometheus-rules/templates/kaas/phoenix/alerting-rules/capa.inhibition.rules.yml

```diff
@@ -0,0 +1,34 @@
+{{- if eq .Values.managementCluster.provider.kind "capa" }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+    cluster_type: "management_cluster"
+  name: capa.inhibitions.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: capa.inhibitions
+    rules:
+    - alert: InhibitionClusterWithoutWorkerNodes
+      annotations:
+        description: '{{`Cluster ({{ $labels.cluster_id }}) has no worker nodes.`}}'
+      expr: |-
+        label_replace(
```

Comment:
Why is this for capa only? otherwise it looks good

Reply:
I would think that with capi_machinepool_spec_replicas and `capi_machinedeployment_spec_replicas` we should be good?

Reply:
Because the metrics is only for CAPA.
Ref: https://gigantic.slack.com/archives/C02HLSDH3DZ/p1729502169569249?thread_ts=1729498455.102279&cid=C02HLSDH3DZ

Reply:
Makes sense :)